resources/outputs.ts

```diff
@@ -39,15 +12,15 @@ const numberToOutputString = (n) => {
     cn: str,
     ko: str,
   };
-};
+}
 
 // General guidelines:
 // * property names should closely match English text
 // * use OnPlayer suffix for things with `${player}`
 // * use OnTarget suffix for things with `${name}`
 // * any other parameters (of which there are none, currently) should use consistent suffixes.
 // * the value of each property should be a single object with localized keys
-const Outputs = new ThrowOnInvalidProxy({
```

Comment:
I wonder if this is worth keeping around until all triggers are converted.

Reply:
It looks tests passed, so `.toJSON()` isn't coveraged yet?

How can I test this 🤔

Reply:
`.toJSON` is just there to support somebody doing `JSON.stringify` on `Outputs`.

This proxy is there to do what typescript will do in the future, in that it validates that any key you call on Outputs is valid.  You can test this by changing any access on Outputs and running npm run test, e.g. change `e12s.js` line 546 `junctionWithCast: Outputs.spread` to `junctionWithCast: Outputs.thisDoesntExist` and then run tests.  You'll get an error.

Once all the trigger files are converted to typescript such that typescript can provide this error, I think this can be safely removed.

Reply:
![image](https://user-images.githubusercontent.com/13553903/108617160-029cc900-744f-11eb-81e0-050b53e69489.png)

![image](https://user-images.githubusercontent.com/13553903/108617179-2233f180-744f-11eb-9546-b8e3a2ccd420.png)

In current tests, I think we could remove this proxy?

Reply:
Oh!  If tests already catch undefined output strings, then that seems good to remove after all.  Sorry!